lib/rubocop/cop/style/redundant_conditional.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks for redundant returning of true/false in conditionals
+      #
+      # @example
+      #   # bad
+      #   x == y ? true : false
+      #
+      #   # bad
+      #   if x == y
+      #     true
+      #   else
+      #     false
+      #   end
+      #
+      #   # good
+      #   x == y
+      #
+      #   # bad
+      #   x == y ? false : true
+      #
+      #   # good
+      #   x != y
+      class RedundantConditional < Cop
+        COMPARISON_OPERATORS = RuboCop::AST::Node::COMPARISON_OPERATORS
+                               .reject { |op| op == :! }.freeze
```

Comment:
I think `:!` has been incorrectly lumped together with comparison operators in the constant in `Node`. It's okay to have a special case for it in your PR. I'll see if I can surgically remove it at a later point. 🙂